minerl/herobraine/hero/handlers/agent/actions/equip.py

```diff
@@ -27,6 +25,22 @@ def __init__(self, items: list, _default='none', _other='other'):
         super().__init__(self._command, self._items, _default=_default, _other=_other),
         self.previous = self._default
 
+    def to_hero(self, x):
+        assert isinstance(x, str)
+
+        # Add a delimiter if necessary and check for problem cases
+        n_delimiter = x.count('#')
+        if n_delimiter == 1:
+            # Validate "adjective" and then send proceed.
+            item_type, metadata_str = x.split('#')
+            assert len(item_type) > 0
+            assert int(metadata_str) in range(16)
+            return super().to_hero(x)
+        elif n_delimiter == 0:
+            return self.to_hero(f"{x}#0")
```

Comment:
Actually, want to send "planks#~" instead. ("planks" will select any planks, rather than only planks#0.)